index.html

```diff
@@ -1,18 +1,53 @@
 <!DOCTYPE html>
 <html lang="pl">
-
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>TITLE</title>
     <link rel="stylesheet" href="styles/main.css" type="text/css" media="all" />
-</head>
+    <link rel="stylesheet" href="styles/basics.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway&display=swap"
```

Comment:
W ramach ciekawostki powiem, że najlepszym jest możliwość importowania tylko wybranych wielkości fontu, dzięki temu nie pobieramy wszystkich fontów a jedynie wybrane co podnosi szybkość działania naszej strony, może nie jest to duża różnica, ale zawsze coś ^^
Nie musimy teraz tutaj tego dodawać :D

Reply:
Dobrze wiedzieć. Znalazłam jak to się robi i zaimportowałam tylko weight 400 i 700, bo to jedyne używane.

Reply:
Extra!